l10n/en/firefox/new/desktop.ftl

```diff
@@ -0,0 +1,155 @@
+firefox-desktop-download-get-firefox-ios = Get { -brand-name-firefox-browser } for <strong>{ -brand-name-ios }</strong>
+firefox-desktop-download-download-the-mobile = Download the { -brand-name-firefox } mobile browser for automatic protection on all your devices.
+firefox-desktop-download-learn-about-the = Learn about the { -brand-name-firefox } desktop browser below
+
+# Variables:
+#   $attrs (attrs) - link to https://addons.mozilla.org/firefox/extensions/
+firefox-desktop-download-from-security-to = From security to news to gaming, there’s an <a { $attrs }>extension for everyone</a>. Add as many as you want until your browser is just right.
+
+# Variables:
+#   $attrs (attrs) - link to https://www.mozilla.org/en-US/about/
+firefox-desktop-download-firefox-was-created = { -brand-name-firefox } was created by <a {$attrs }>{ -brand-name-mozilla }</a> as a faster, more private alternative to browsers like Internet Explorer, and now { -brand-name-chrome }. Today, our mission-driven company and volunteer community continue to put your privacy above all else.
+
+# Variables:
+#   $attrs (attrs) - link to https://www.mozilla.org/en-US/firefox/privacy/
+firefox-desktop-download-as-the-internet = As the internet grows and changes, { -brand-name-firefox } continues to focus on your right to privacy  — we call it the <a { $attrs }>Personal Data Promise</a>: Take less. Keep it safe. No secrets. Your data, your web activity, your life online is protected with { -brand-name-firefox }.
+
+# Variables:
+#   $attrs (attrs) - link to https://accounts.firefox.com/signin
+firefox-desktop-download-firefox-is-available = { -brand-name-firefox } is available on all your devices; take your tabs, history and bookmarks with you. All you need is a <a { $attrs }>{ -brand-name-firefox } account</a>.
+
+# Variables:
+#   $attrs (attrs) - link to https://support.mozilla.org/en-US/kb/firefox-screenshots
+firefox-desktop-download-grab-a-high = Grab a high-resolution image of anything online with our <a { $attrs }>screenshot</a> tool built right in the browser
+
+# Variables:
+#   $attrs (attrs) - link to https://addons.mozilla.org/firefox/extensions/
+firefox-desktop-download-firefox-automatically = { -brand-name-firefox } automatically <a { $attrs }>blocks many third party trackers</a> from collecting and selling your web activity.
+
+# Variables:
+#   $attrs (attrs) - link to https://addons.mozilla.org/firefox/extensions/
+firefox-desktop-download-questions = Questions? <a { $attrs }>{ -brand-name-mozilla } support</a> has you covered.
+
+
+## URL: https://www-dev.allizom.org/firefox/new/thanks/
+
+firefox-desktop-download-now-finish = Almost there! Now finish <strong>installing</strong> { -brand-name-firefox }.
+firefox-desktop-download-click-installer = Click on the { -brand-name-firefox } Installer
+firefox-desktop-download-click-installer-bottom = Click the { -brand-name-firefox } installer in the bottom left corner of your browser.
```

Comment:
Hmm - "bottom left corner of your browser" isn't something that's specific to Windows as an OS, is it? It varies browser to browser. Even in Chrome, if someone is using a RTL browser this would be bottom right?

Is there something more generic we could use, that doesn't assume the browser you're using?

Reply:
✅ We vague-ified the install instructions and removed the images.